deploy/kubernetes/resources/chirper/activity-stream-impl-statefulset.json

```diff
@@ -19,6 +19,11 @@
             "name": "activityservice",
             "image": "chirper/activity-stream-impl",
             "imagePullPolicy": "Never",
+            "resources": {
+              "requests": {
+                "memory": "512Mi"
```

Comment:
Tip: Use `pmap` to profile - you may get away with considerably less than that.

Reply:
Thanks - raised as #97